src/index.html

```diff
@@ -17,16 +17,33 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+      method="get"
+    >
+      <input
+        class="search search--big"
+        name="search-big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <input
+        class="search search--small"
```

Comment:
The `class` attribute value `search search--small` seems a bit confusing. If `search` is a block, and `small` is its modifier, it would be more appropriate to name the class `search search--small`.